ci_config.json

```diff
@@ -254,6 +254,12 @@
       "libboost-iostreams-dev"
     ]
   },
+  "pcre2": {
+    "_comment": "Tests are broken on Windows",
+    "build_options": [
+      "pcre2:test=false"
```

Comment:
NACK

Reply:
> There's also a RunGrepTest. Wonder if that should be added...

You added another test, the test that was already there works, but the one you added fails on Windows. The solution to this isn't to then disable all tests for all platforms.

Simply mark this one as `should_fail: true` on that platform.

Reply:
I think this should do it.

Reply:
Yep, thanks. :)